scripts/Logistics-Pipes.zs

```diff
@@ -0,0 +1,873 @@
+[null, <ore:craftingToolScrewdriver>, null]]);
+
+// --- Disconnection Upgrade (EAST)
+// - Alternative Recipe
+recipes.addShaped(DisconnectEast, [
+[null, null, null],
+[null, DisconnectDown, <ore:craftingToolScrewdriver>],
+[null, null, null]]);
+// -
+recipes.addShaped(DisconnectEast, [
+[null, null, null],
+[null, DisconnectNorth, <ore:craftingToolScrewdriver>],
+[null, null, null]]);
+// -
+recipes.addShaped(DisconnectEast, [
+[null, null, null],
+[null, DisconnectSouth, <ore:craftingToolScrewdriver>],
+[null, null, null]]);
+// -
+recipes.addShaped(DisconnectEast, [
+[null, null, null],
+[null, DisconnectUp, <ore:craftingToolScrewdriver>],
+[null, null, null]]);
+// -
+recipes.addShaped(DisconnectEast, [
+[null, null, null],
+[null, DisconnectWest, <ore:craftingToolScrewdriver>],
+[null, null, null]]);
+
+// --- Disconnection Upgrade (WEST)
+// - Alternative Recipe
+recipes.addShaped(DisconnectWest, [
+[null, null, null],
+[<ore:craftingToolScrewdriver>, DisconnectDown, null],
+[null, null, null]]);
+// -
+recipes.addShaped(DisconnectWest, [
+[null, null, null],
+[<ore:craftingToolScrewdriver>, DisconnectNorth, null],
+[null, null, null]]);
+// -
+recipes.addShaped(DisconnectWest, [
+[null, null, null],
+[<ore:craftingToolScrewdriver>, DisconnectSouth, null],
+[null, null, null]]);
+// -
+recipes.addShaped(DisconnectWest, [
+[null, null, null],
+[<ore:craftingToolScrewdriver>, DisconnectEast, null],
+[null, null, null]]);
+// -
+recipes.addShaped(DisconnectWest, [
+[null, null, null],
+[<ore:craftingToolScrewdriver>, DisconnectUp, null],
+[null, null, null]]);
+
+// ||||| Fluid Pipes |||||
+//Disabled for right now!
+
+// #======= Hiding Stuff =======#
+
+NEI.hide(<LogisticsPipes:item.pipeComponents:*>);
+NEI.hide(<LogisticsPipes:item.itemUpgrade:44>);
+NEI.hide(<LogisticsPipes:logisticsSolidBlock:0>);
+NEI.hide(<LogisticsPipes:logisticsSolidBlock:11>);
+NEI.hide(<LogisticsPipes:logisticsSolidBlock:12>);
+NEI.hide(<LogisticsPipes:logisticsPipeBlock>);
+NEI.hide(<LogisticsPipes:item.logisticsParts:3>);
+NEI.hide(<LogisticsPipes:item.itemUpgrade:30>);
+NEI.hide(<LogisticsPipes:item.itemUpgrade:32>);
+NEI.hide(<LogisticsPipes:item.itemUpgrade:33>);
+NEI.hide(<LogisticsPipes:item.itemUpgrade:34>);
+NEI.hide(<LogisticsPipes:item.itemUpgrade:35>);
+NEI.hide(<LogisticsPipes:item.itemUpgrade:36>);
+NEI.hide(<LogisticsPipes:item.itemModule:301>);
+
+// Hidden until added through code, 
+// Theese are used to connect 2 LP networks through chests and might lag, they are disabled for now
+NEI.hide(<LogisticsPipes:item.PipeItemsInvSysConnector>);
+NEI.hide(<LogisticsPipes:item.PipeItemsSystemEntranceLogistics>);
+NEI.hide(<LogisticsPipes:item.PipeItemsSystemDestinationLogistics>);
+NEI.hide(<LogisticsPipes:item.PipeItemsFirewall>);
+NEI.hide(<LogisticsPipes:item.logisticsItemCard>); // this item can't be easily added through Minetweaker
+//
+
+// Fluid pipes are too op according to me, But if you really want them enabled, i can re-enable them.

```

Comment:
Please do. The ability to maintain fluid level in a compact way is really nice.

Reply:
I will think up some recipes for theese later, but ok i will add them